upgrade-binaries/README.md

```diff
@@ -0,0 +1,12 @@
+# Static binaries for 1.x to 2.x upgrades
+
+Some additional tools are required for upgrading a 1.x resinOS prior to 1.27 to 2.x. These are:
+
+ * e2label
+ * mkfs.ext4
+ * resize2fs
+ * tar
+
+These binaries have been built with buildroot, which makes static builds much easier than Yocto.
```

Comment:
The key points are:

```
BR2_STATIC_LIBS=y
# BR2_SHARED_LIBS is not set
BR2_TOOLCHAIN=y
BR2_TOOLCHAIN_USES_MUSL=y
BR2_TOOLCHAIN_BUILDROOT=y

BR2_PACKAGE_E2FSPROGS=y
BR2_PACKAGE_E2FSPROGS_E2LABEL=y
BR2_PACKAGE_E2FSPROGS_MKE2FS=y
BR2_PACKAGE_E2FSPROGS_RESIZE2FS=y

BR2_PACKAGE_TAR=y
```

[static-config.txt](https://github.com/resin-os/resinhup/files/1139554/static-config.txt)